setup.py

```diff
@@ -246,7 +246,8 @@ def check_cuda_torch_binary_vs_bare_metal(cuda_dir):
                                    'apex/contrib/csrc/groupbn/ipc.cu',
                                    'apex/contrib/csrc/groupbn/interface.cpp',
                                    'apex/contrib/csrc/groupbn/batch_norm_add_relu.cu'],
-                          include_dirs=[os.path.join(this_dir, 'csrc')],
+                          include_dirs=[os.path.join(this_dir, 'csrc'),
+                                        os.path.join(this_dir, 'apex/contrib/csrc/groupbn')],
                           extra_compile_args={'cxx': [] + version_dependent_macros,
```

Comment:
@sarunyap This shouldn't be needed as hipify will automatically search in the current directory of the source file for the header?

Reply:
I tried building APEX without this and I ran into a build problem.  So, I think it's needed.

Reply:
@sarunyap I just merged #53 which works around an ordering issue during hipification of the various Apex extensions. Would you kindly rebase/merge your changes and try without the above entry to see if you still get a build issue?

Reply:
@jithunnair-amd I tried this but it still gives me error

Reply:
@sarunyap I dug into this, and you are right that the hipify logic doesn't look into the current directory of the `'apex/contrib/csrc/groupbn/batch_norm_add_relu.cu'` file for , so it doesn't find `batch_norm_add_relu.h` unless we explicitly add the `'apex/contrib/csrc/groupbn'` path to the `include_dirs`. This is actually a known bug in hipify where the `include_dirs` argument in setup.py call is erroneously passed in as the `includes` argument to hipify call. We'll need to fix this hipify bug in upstream pytorch. Until then, the change above should work.